README.md

```diff
@@ -1,47 +1,48 @@
-# Event Structures Formalization
+# Mechanized Theory of Event Structures
 
 ![GitHub Actions][github-actions-badge]
 
 [github-actions-badge]: https://github.com/volodeyka/event-struct/workflows/CI/badge.svg
 
-A library of formalized event structure theory applied to weak-memory models.
+A Coq library of formalized event structure theory with applications to concurrency semantics.
 Includes notion and theory of execution event structures, labeled transition
 system on execution event structures, big-step semantics for simple
 register machine, and well-founded relations theory.
 
-## Files in poject
-All files include description of its constituents.
+## Description of Files
```

Comment:
Grammar nits: this section lacks a number of articles ("a" and/or "the" mostly). While it's fine to skip them in section titles but I'd prefer having them in the text body.

Reply:
Oh, I have some problems with them, but I'll try to fix it

Reply:
Oh, ok, I thought that was a stylistic choice :)